### PR TITLE
Permission issues with DB2 example image

### PR DESCRIPTION
**Pull request by jcechace**
- More descriptive message after cdc setup completes
- Modified file permissions to match config from https://github.com/debezium/debezium-connector-db2/blob/main/src/test/docker/db2-cdc-docker/Dockerfile
- Using fixed docker tag

### CODE REVIEW

File: tutorial/debezium-db2-init/db2server/Dockerfile [hunk 1]
```diff
@@ -13,11 +13,11 @@ ADD startup-cdc-demo.sql /asncdctools/src
...
-RUN chmod -R  777  /asncdctools
-
-RUN mkdir /var/custom
-RUN chmod -R  777 /var/custom
+RUN mkdir /var/custom && \
+    chmod -R  777 /asncdctools && \
+    chmod -R  777 /var/custom
 
 ADD cdcsetup.sh /var/custom
-ADD custom-init /var/custom-init
```
Comment (jpechane) [inline tutorial/debezium-db2-init/db2server/Dockerfile:22]: Why is this dir no longer added?

Comment (jcechace) [reply tutorial/debezium-db2-init/db2server/Dockerfile:22]: Mistake, fixed!

File: tutorial/debezium-db2-init/db2server/Dockerfile [hunk 2]
```diff
@@ -13,11 +13,15 @@ ADD startup-cdc-demo.sql /asncdctools/src
...
+    chmod -R  777 /asncdctools && \
+    chmod -R  777 /var/custom
 
 ADD cdcsetup.sh /var/custom
 ADD custom-init /var/custom-init
-ADD openshift_entrypoint.sh /var/db2_setup/lib/
+
+RUN chmod -R 777 /var/custom-init
+
+ADD openshift_entrypoint.sh /var/db2_setup/lib
```
Comment (jpechane) [inline tutorial/debezium-db2-init/db2server/Dockerfile:25]: Maybe you should change the privileges here too?

**Comment by jpechane** [comment]
@jcechace Applied, thanks

**Pull request merged**